Dev

steveri opened the pull request:
Time for our annual dev->master merge :)

alexcarsello commented:
Looking through this... Do you have any idea why commits from June are showing up in this PR? These have already been merged into master.

steveri commented:
No I do *not* know why there are commits from last June.
And yes it is a little bit concerning :(

I'm very confident about my changes, which account for most of the 81 changed files, but you might want to double-check anything from your domain e.g.

* deleted file hardware/generator_z/global_controller/analog_regfile.vp
* changed file hardware/generator_z/memory_tile/memory_tile.vp
* changed file hardware/generator_z/top/ncsim_verif/JTAGDriver.svp

...maybe a couple of others...?

steveri commented:
> No I do not know why there are commits from last June.

Actually...could this be related to the fact that, after doing a dev->master merge, I/we don't delete dev but continue altering it such that, when the next merge happens, all the old history is still there to be dealt with...?
